packages/scenario-harness/ui/app.js

```
@@ -485,6 +483,20 @@ function readReadbackSnapshot() {
 
+// Server-side translator turns the snapshot into a telemetry patch and
+// `tickOnce`s the mock controller, so the production readback resolver
+// (`getReadbackSnapshot()` from sim-events-iracing) sees the user's
+// selections when the readback fires. Per-control change listeners call
+// this on every edit so iRacing's per-toggle confirmations also fire as
+// the user composes — disconnect the controller if you want silence.
+async function pushReadbackSnapshot() {
+  try {
+    await post("/api/readback/snapshot", readReadbackSnapshot());
+  } catch (e) {
+    console.error("Readback snapshot push failed:", e);
```

ESLint

[error] 'console' is not defined.

(no-undef)